packages/mock-block-dock/src/useMockDatastore.ts

```diff
@@ -44,16 +46,38 @@ export const useMockDatastore: UseMockDataStore = (
     entities: [],
     links: [],
     entityTypes: [],
+    blockEntityId: "",
   },
 ) => {
   const [entities, setEntities] = useState<MockDataStore["entities"]>(
     initialData.entities,
   );
+  const [latestBlockEntity, setLatestBlockEntity] = useState(
```

Comment:
Moved `latestBlockEntity` to `useMockDataStore` since it was very tightly dependent on the values being passed to, and returned from it.

Reply:
I think this makes the purpose `useMockDatastore` less well defined - it would be better if we kept it to returning data and functions to manipulate the data, and leave it up to consumers to do what they want with the data (e.g. find a certain entity in it) - i.e. leave this calculation outside it. Then `useMockDatastore` doesn't have to know about blocks or `latestBlockEntity`.